docs/Community/sig-website/README.md

```diff
@@ -0,0 +1,30 @@
+Website Special Interest Group
+=============================
+
+SIG-Website is responsible for ...
```

Comment:
"updating and maintaining https://openenclave.io" presumably?

Reply:
I would presume so yes, but I'd prefer that the sig lead fill that in, in a PR, rather than me doing so.